conf/evolutions/default/1.sql

```diff
@@ -75,8 +75,16 @@ CREATE TABLE IF NOT EXISTS restaurant_photo (
 	photo_path character varying(256) NOT NULL
 );
 
+CREATE TABLE IF NOT EXISTS activity_log (
```

Comment:
You need to add new SQL statement to new file (3.sql). CREATE TABLE command would never be executed because step (1) is already completed.